client-v2/src/components/FrontsEdit/Edit.tsx

```diff
@@ -42,16 +45,32 @@ const FrontsEditContainer = styled('div')`
 `;
 
 const SingleFrontContainer = styled('div')`
+  flex: 1 1 auto;
   height: 100%;
+  min-width: 626px;
+`;
+
+// This is just to stop the feed / clipboard from filling the screen when no fronts
+// are selected
+const NoFrontContainer = styled.div`
```

Comment:
Would limiting the maximum width of the feed/clipboard achieve the same thing without markup? (I've probably missed something!)

Reply:
I did try setting max-width here to 50% and I can't remember what broke ... I'm going to sit here and try and remember.

*time elapses*

Figured it ... it's because we cant' guarantee what we actually want the max-width to be. On sufficiently narrow screens it's over 50% of the viewport but actually we rely on the content on the right to squash us down to (hopefully) 50% when a front pops in thanks to the flex layout.